pyregion/__init__.py

```diff
@@ -9,12 +9,7 @@
 a new astronomy package for regions based on Astropy.
 """
 
-# Affiliated packages may add whatever they like to this file, but
-# should keep this content at the top.
-# ----------------------------------------------------------------------------
-from ._astropy_init import *
```

Comment:
So you don't need the test runner from `_astropy_init`?

Reply:
I mean, you removed a user's ability to test the installed package by running `import pyregion; pyregion.test()`. Is that intentional?

Reply:
Yes, intentional.  Is there a reason to support this very non-standard way of running tests?

If there is, I can put it back, but I think it's very much a legacy piece of infrastructure that is not common or recommended in python outside of `astropy`.

Reply:
I do find it nice that user can run tests on their *installed* copy. I still call that myself when I publish though I guess I don't have to but it is just so convenient.

https://github.com/spacetelescope/synphot_refactor/blob/cb3ff4b05ae5f20419b5b4a781eac9496f24936c/.github/workflows/publish-to-pypi.yml#L38

Here, you are actually removing a feature, so should get okay from @larrybradley or @keflavich first.

Reply:
Running tests on installed copies is very easy:

```
pip install pyregion[test]
pytest --pyargs pyregion
```

And is the recommend way.  I even updated the docs to do this. This ensures test dependencies are installed.

One can also run tests on the installed version if you are in a git repo checkout doing development:

```
pip install .[test]
pytest --pyargs pyregion --import-mode=importlib
```

Which will install the package in site-packages and then run the tests from in there as well, ensuring not to add `.` to sys.path.  This is more convoluted, but it is the way forward for developers using `pytest`.

Reply:
Ah, so `pytest --pyargs pyregion` can detect the installed copy even if you didn't check out the source or not in the site-packages dir? I didn't know that. pytest sure has come a long way since that time the whole thing was in one file that astropy bundles.